components/sup/src/util/users.rs

```diff
@@ -33,6 +33,10 @@ fn check_pkg_user_and_group(pkg_install: &PackageInstall) -> Result<Option<(Stri
     let svc_group = try!(pkg_install.svc_group());
     match (svc_user, svc_group) {
         (Some(user), Some(group)) => {
+            // a package has a SVC_USER and SVC_GROUP defined,
+            // these MUST exist in order to continue
+            debug!("SVC_USER = {}", &user);
```

Comment:
oh weird. this must have been something off wit hmy commit because I did not add these lines. I'm gonna fix this commit!